app/handlers/handlers_resource.py

```diff
@@ -3,20 +3,27 @@
 import kopf
 
 from app.api import TwingateAPIClient
-from app.crds import K8sMetadata, ResourceSpec
+from app.crds import ResourceSpec
 from app.handlers.base import fail, success
 
 
 @kopf.on.create("twingateresource")
-def twingate_resource_create(body, meta, spec, memo, logger, patch, **kwargs):
-    logger.info("Got a create request: %s. Metadata: %s", spec, meta)
-    k8s_metadata = K8sMetadata(**meta)
+def twingate_resource_create(body, labels, spec, memo, logger, patch, **kwargs):
+    logger.info("Got a create request: %s. Labels: %s", spec, labels)
     resource = ResourceSpec(**spec)
     client = TwingateAPIClient(memo.twingate_settings, logger=logger)
+    labels = {
+        **{
```

Comment:
nit: I'd put this in the settings class as `memo.twingate_settings.default_resource_tags_dict`

Reply:
I also like the `|` opeartion better than ** creation:
```
memo.twingate_settings.default_resource_tags_dict | dict(labels)
```

Reply:
Yeah, there are a few other places that could be simplified using dict union `|` operator. We should update them too.

Reply:
We could keep `default_resource_tags` Pydantic field definition as is and create a property that derives from the field e.g.
```python
class TwingateOperatorSettings(BaseSettings):
    default_resource_tags: list[dict[str, str]] = []

    @property
    def default_resource_k8s_labels() -> dict:
        return {
            tag["name"]: tag["value"]
            for tag in self.default_resource_tags
        }
```